pymeasure/instruments/keithley/keithley6517b.py

```diff
@@ -347,3 +347,4 @@ def shutdown(self):
         self.ramp_to_voltage(0.0)
         self.stop_buffer()
         self.disable_source()
```

Comment:
Log message could be removed.
(I will not comment about this for the remaining files)